apps/05Mastermind.js

```diff
@@ -26,27 +28,64 @@ function getRandomInt(min, max) {
   return Math.floor(Math.random() * (max - min)) + min;
 }
 
-function generateHint() {
-    // your code here
+function generateHint(solution, gess) {
+
+    var solutionArray = [];
+    var guessArray = [];
+    var correctLetterLocations = 0;
+    var correctLetters = 0;
+    var targetIndex = 0;
+
+    solutionArray = solution.split("");
+    guessArray = gess.split("");
+    for (var i=0; i<4;i++){
+      if (guessArray[i] === solutionArray[i]){
+        correctLetterLocations++;
+        solutionArray[i] = null;
+        guessArray[i] = '';
```

Comment:
Interesting choice! Why null in the solutionArray and '' in the guessArray?

(I see that it works, nice job)